default.nix

```diff
@@ -53,7 +53,23 @@ let
     buildInputs = [ agdaStdlib agdaStdlibClasses ];
   };
 
-  deps = [ agdaStdlib agdaStdlibClasses agdaStdlibMeta ];
+  agdaSets = customAgda.agdaPackages.mkDerivation {
+    inherit (locales) LANG LC_ALL LOCALE_ARCHIVE;
+    pname = "agda-sets";
+    version = "2.1.1";
+    src = fetchFromGitHub {
+      repo = "agda-sets";
+      owner = "input-output-hk";
```

Comment:
Just curious why the `agda-sets` repo is owned by input-output-hk, while the formal spec repo is with intersect.

Reply:
I don't think I can make a repository on the Intersect repo, but I'm also not super sure where this repo should live, so I've put it there for now.